src/scripts/generate-imported-targets-from-snyk.ts

```diff
@@ -40,6 +40,16 @@ export function projectToTarget(
     name,
   };
 }
+export function bitbucketServerProjectToTarget(
+  project: Pick<SnykProject, 'name' | 'branch'>,
+): Target {
+  const [projectKey, repoSlug] = project.name.split(':')[0].split('/');
+  return {
+    projectKey,
+    repoSlug,
+    branch: project.branch || undefined,
```

Comment:
After looking at the `import:data` PR we never get back the project branch so it won't be in the logged target right? Will need to see the `imported-target` log for BBS to confirm. 

I think we will need to drop this for BBS because the API does not support branches :(